test_agent/python/testagent.py

```diff
@@ -128,22 +167,36 @@ def handle_unregister_listener_command(json_msg):
 def handle_invoke_method_command(json_msg):
     uri = dict_to_proto(json_msg["data"], UUri())
     payload = dict_to_proto(json_msg["data"]["payload"], UPayload())
-    res_future = transport.invoke_method(uri, payload, CallOptions(ttl=10000))
+    res_future: Future = transport.invoke_method(uri, payload, CallOptions(ttl=10000))
 
     def handle_response(message):
-        message = message.result()
-        send_to_test_manager(message, CONSTANTS.RESPONSE_RPC)
+        message: Message = message.result()
+        send_to_test_manager(message, CONSTANTS.INVOKE_METHOD_COMMAND, received_test_id=json_msg["test_id"])
 
     res_future.add_done_callback(handle_response)
 
 
-def handle_uri_serialize_command(json_msg):
-    uri = dict_to_proto(json_msg["data"], UUri())
-    send_to_test_manager(LongUriSerializer().serialize(uri), CONSTANTS.SERIALIZE_URI)
+def send_longserialize_uuri(json_msg: Dict[str, Any]):
+    uri: UUri = dict_to_proto(json_msg["data"], UUri())
+    serialized_uuri: str = LongUriSerializer().serialize(uri)
+    send_to_test_manager(serialized_uuri, CONSTANTS.SERIALIZE_URI, received_test_id=json_msg["test_id"])
 
 
-def handle_uri_deserialize_command(json_msg):
-    send_to_test_manager(LongUriSerializer().deserialize(json_msg["data"]), CONSTANTS.DESERIALIZE_URI)
+def send_longdeserialize_uri(json_msg: Dict[str, Any]):
+    uuri: UUri = LongUriSerializer().deserialize(json_msg["data"])
+    send_to_test_manager(uuri, CONSTANTS.DESERIALIZE_URI, received_test_id=json_msg["test_id"])
+
+
+def send_longdeserialize_uuid(json_msg: Dict[str, Any]):
+    uuid: UUID = LongUuidSerializer().deserialize(json_msg["data"])
+    send_to_test_manager(uuid, CONSTANTS.DESERIALIZE_UUID, received_test_id=json_msg["test_id"])
+
+
+def send_longserialize_uuid(json_msg: Dict[str, Any]):
```

Comment:
Either rename it to handle_long_serialize_uuid_command to make it consistent with others command handlers or put _ in between long and serialize.

Reply:
will do this